packages/config/config/devices/0x0234/zdb5100.json

```diff
@@ -1,827 +1,557 @@
 		},
 		"3": {
-			"label": "Duration of on/off.",
-			"description": "This parameter specifies the duration when turning the light on or off.",
-			"valueSize": 1,
-			"minValue": 0,
-			"maxValue": 255,
-			"defaultValue": 0,
-			"unsigned": true,
-			"readOnly": false,
-			"writeOnly": false,
-			"allowManualEntry": true
+			"$import": "templates/logic_group_template.json#duration_on_off"
 		},
 		"4": {
-			"label": "Dimmer mode.",
-			"description": "The dimmer can work in different modes.",
-			"valueSize": 1,
-			"minValue": 0,
-			"maxValue": 2,
-			"defaultValue": 1,
-			"readOnly": false,
-			"writeOnly": false,
-			"allowManualEntry": false,
-			"options": [
-				{
-					"label": "No dimming, only on/off (0%/100%).",
-					"value": 0
-				},
-				{
-					"label": "Trailing edge dimming.",
-					"value": 1
-				},
-				{
-					"label": "Leading edge dimming.",
-					"value": 2
-				}
-			]
+			"$import": "templates/logic_group_template.json#dimmer_mode"
 		},
 		"5": {
-			"label": "Dimmer minimum level.",
-			"description": "This parameter specifies the actual level of the dimmer output when set to 0%.",
-			"valueSize": 1,
-			"minValue": 0,
-			"maxValue": 99,
-			"defaultValue": 0,
-			"readOnly": false,
-			"writeOnly": false,
-			"allowManualEntry": true
+			"$import": "templates/logic_group_template.json#dimmer_min"
 		},
 		"6": {
-			"label": "Dimmer maximum level.",
-			"description": "This parameter specifies the actual level of the dimmer output when set to 99%",
-			"valueSize": 1,
-			"minValue": 1,
-			"maxValue": 99,
-			"defaultValue": 99,
-			"readOnly": false,
-			"writeOnly": false,
-			"allowManualEntry": true
+			"$import": "templates/logic_group_template.json#dimmer_max"
 		},
 		"7": {
-			"label": "Central Scene notifications",
-			"description": "This parameter can be used for disabling Central Scene notifications.",
-			"valueSize": 1,
-			"minValue": 0,
-			"maxValue": 1,
-			"defaultValue": 1,
-			"readOnly": false,
-			"writeOnly": false,
-			"allowManualEntry": false,
-			"options": [
-				{
-					"label": "Notifications are disabled.",
-					"value": 0
-				},
-				{
-					"label": "Notifications are enabled.",
-					"value": 1
-				}
-			]
+			"$import": "templates/logic_group_template.json#enable_central_scene"
 		},
 		"8": {
-			"label": "Double-activation functionality",
-			"description": "Specifies the reaction when double-activating the pushbuttons",
-			"valueSize": 1,
-			"minValue": 0,
-			"maxValue": 1,
-			"defaultValue": 0,
-			"readOnly": false,
-			"writeOnly": false,
-			"allowManualEntry": false,
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Double-Activation Functionality",
+			"defaultValue": 1,
 			"options": [
 				{
-					"label": "Double-activation disabled.",
+					"label": "Disabled",
 					"value": 0
 				},
 				{
-					"label": "Double-activation sets light to 100%.",
+					"label": "Sets light to 100%",
```

Comment:
If the only options are Disabled and Sets to 100%, I'd just make this Enable/Disable.